tests/unit/fem/test_get_value.cpp

```diff
@@ -338,17 +327,13 @@ TEST_CASE("2D GetValue",
                double dgv_err = 0.0;
                double dgi_err = 0.0;
 
-               double tip_data[dim];
-               Vector tip(tip_data, dim);
```

Comment:
This change should also fix an issue that I noticed was introduced previously in `test_get_values.cpp`:

`double tip_data[dim];` would fail with a compilation error when compiling with `-pedantic-errors` enabled: `error: variable length arrays are a C99 feature`.

This could be fixed by e.g. marking `dim` as `constexpr` (or just removing `int tip_data[dim]` also works)...

Perhaps we should change our default compilation flags to include `-pedantic-errors`?

Reply:
More generally, we can add `-Werror` (along with `-pedantic`, `-Wall`, etc) to travis to catch all warnings by turning them into errors. However, it will be best to do this in a separate PR.

Reply:
Agreed. Also, for this particular issue, `-Wall` doesn't give a warning, so `-pedantic` would be required (`-Wextra` will probably result in too many warnings).

Reply:
Yes, `-pedantic` and `-Wall` are independent (I use them both in my `user.mk` config file). And, yes, `-Wextra` will give too many warnings, e.g. for unused parameters -- we have a bunch of those.